Sources/JWTKeychain/Support/Configuration/Configuration.swift

```diff
@@ -94,55 +130,90 @@ public struct Configuration: ConfigurationType {
             return ES512(key: key)
         default:
             return HS256(key: key)
-            
+
         }
-        
+
     }
-    
-    
+
+
     /// Validates a given token
     ///
     /// - Parameter token: string with the token
     /// - Returns: true if token is valid, else false
     /// - Throws: if unable to create JWT instance
-    public static func validateToken(token: String) throws -> Bool {
-        
+    public func validateToken(token: String) throws -> Bool {
+
         do {
-            
+
             // Validate our current access token
             let receivedJWT = try JWT(token: token)
-            
-            var key: Bytes = Configuration.getTokenSignatureKey()
-            
-            if(Configuration.instance!.publicKey != nil){
-                
-                key = try Configuration.getTokenPublicKey()
+
+            var key: Bytes = self.getTokenSignatureKey()
+
+            if(self.publicKey != nil){
+
+                key = try self.getTokenPublicKey()
             }
-            
+
             // Verify signature
-            let signer: Signer = Configuration.getSigner(key: key)
+            let signer: Signer = self.getSigner(key: key)
             if try receivedJWT.verifySignatureWith(signer) {
-                
-                
+
+
                 // If we have expiration set on config, verify it
-                if Configuration.instance!.secondsToExpire! > 0 {
-                    
+                if self.secondsToExpire! > 0 {
```

Comment:
`secondsToExpire` should no longer be optional and the force unwrap should be removed.